ports/esp32/boards/TINYPICO/modules/tinypico.py

```diff
@@ -0,0 +1,136 @@
+# TinyPICO MicroPython Helper Library
+# 2019 Seon Rozenblum, Matt Trentini
+#
+# Project home:
+#   https://github.com/TinyPICO
+#
+# 2019-Mar-12 - v0.1 - Initial implementation
+# 2019-May-20 - v1.0 - Initial Release
+# 2019-Sep-01 - v1.1 - Frozen in MicroPython TinyPICO build
+
+"""
+`tinypico` - MicroPython TinyPICO Helper Library
+====================================================
+
+* Author(s): Seon Rozenblum, Matt Trentini
+"""
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/TinyPICO/tinypico-micropython"
+
+# Import required libraries 
+from micropython import const
+from machine import Pin, SPI, ADC
+import machine, time, esp32
+
+# TinyPICO Hardare Pin Assingments
```

Comment:
s/Hardare/Hardware
s/Assingments/Assignments

Reply:
Wow I missed a lot of these.